data/functions.sql

```diff
@@ -585,6 +601,38 @@ BEGIN
 END;
 $$ LANGUAGE plpgsql STABLE;
 
+CREATE OR REPLACE FUNCTION mz_cycling_network_(way_tags hstore, osm_id bigint)
+RETURNS text AS $$
+DECLARE
+  networks text[] := ARRAY(
+    SELECT hstore(tags)->'network'
+    FROM planet_osm_rels
+    WHERE parts && ARRAY[osm_id]
+      AND parts[way_off+1:rel_off] && ARRAY[osm_id]
+      AND mz_is_path_major_route_relation(hstore(tags)));
+BEGIN
+  RETURN CASE
+    WHEN networks && ARRAY['icn'] THEN 'iwn'
```

Comment:
It should, nice catch! :zap: Fixed in ae0b48d.

As you can tell, I don't have tests for the case where we pull the network from relations yet :)